includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -1808,10 +1813,13 @@ private function ampify_ruleset_selectors( $ruleset ) {
 					if ( ! $count ) {
 						continue;
 					}
+					$selectors[]   = $edited_selector . '>' . $html_selector;
 					$replacements += $count;
 					while ( ! empty( $amp_selectors ) ) { // Note: This array contains only a couple items.
 						$amp_selector       = array_shift( $amp_selectors );
-						$edited_selectors[] = preg_replace( $html_pattern, $amp_selector, $original_selector, -1, $count );
+						$edited_selector    = preg_replace( $html_pattern, $amp_selector, $original_selector, -1, $count );
+						$edited_selectors[] = $edited_selector;
```

Comment:
Started thinking that maybe we should only replace it with `amp-img>img` and not `img` at all (same with others)? Adding two selectors would add the same style to both `amp-img` and to `img`, could that cause issues? Also, wondering if in any case we would always need the style for the child element (`img`) and not the `amp-img` at all. Would we need it for the `amp-img` in some cases? Thoughts?

In this case we could actually just replace the child selector rules, this would also solve the issue for `AMP_Playbuzz` where `amp-playbuzz>.pb_feed` would never exist.